### PR TITLE
Fix class namespace

### PR DESCRIPTION
**Pull request by khvalov**
Fix for error:  exception 'yii\base\ErrorException' with message 'Class 'yii\filters\Exception' not found' in /www/vendor/yiisoft/yii2/filters/Cors.php:171

| Q             | A
| ------------- | ---
| Is bugfix?    | yes
| New feature?  | no
| Breaks BC?    | no
| Tests pass?   | yes

### CODE REVIEW

File: framework/filters/Cors.php [hunk 1]
```diff
@@ -168,7 +168,7 @@ public function prepareHeaders($requestHeaders)
                 // Per CORS standard (https://fetch.spec.whatwg.org), wildcard origins shouldn't be used together with credentials
                 if (isset($this->cors['Access-Control-Allow-Credentials']) && $this->cors['Access-Control-Allow-Credentials']) {
                     if (YII_DEBUG) {
-                        throw new Exception("Allowing credentials for wildcard origins is insecure. Please specify more restrictive origins or set 'credentials' to false in your CORS configuration.");
+                        throw new \Exception("Allowing credentials for wildcard origins is insecure. Please specify more restrictive origins or set 'credentials' to false in your CORS configuration.");
```
Comment (samdark) [inline framework/filters/Cors.php]: https://github.com/yiisoft/yii2/blob/master/framework/base/InvalidConfigException.php should be thrown.

**Comment by khvalov** [comment]
Updated to InvalidConfigException

**Review by samdark — APPROVED** [review]
Need a line for changelog. Then merging it right away.

**Comment by khvalov** [comment]
misconfigured CORS filter throws InvalidConfigException in Debug mode instead of native PHP Exception

**Comment by samdark** [comment]
Merged. Thank you!

**Pull request merged**